src/gui/InventoryWindowController.java

```diff
@@ -41,8 +41,8 @@ public void initialize(URL url, ResourceBundle rb) {
      */
     @FXML
     private void onUseClicked(ActionEvent event) {
-        game.command("use", listView.getSelectionModel().getSelectedItem().replace(" ", "-"));
-        addItemsToViewList();
+            game.command("use", listViewSelection(listView));
+            addItemsToViewList();
```

Comment:
Indentation

Reply:
fixed-in: 00432b3a4c70b92bf2e0734f218be5e4bb834768